### PR TITLE
update README to decribe kernel chaos and merge daemonset.yaml

### PR DESCRIPTION
**Pull request by ethercflow**
### What problem does this PR solve? 
- Update README to describe kernel
- Merge two daemonset's deploy config
### Check List 

Tests 

 - Manual test
 - No code

Code changes

 - No code

Side effects

 - None

Related changes

 - None

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: doc/kernel_chaos.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
...
+callchain and frequency. This feature requires 4.18 kernel or above with
+`CONFIG_BPF_KPROBE_OVERRIDE=y`. To enable it, you need to chage `create` value
+to `true`:
+
+```yaml
+bpfki:
+  create: true
+```
+
+in ![values.yaml](../helm/chaos-mesh/values.yaml).
```
Comment (cwen0) [inline doc/kernel_chaos.md]: remove "!"

File: doc/kernel_chaos.md [hunk 2]
```diff
@@ -2,6 +2,21 @@
 
 This document describe how to add kernel chaos experiments in Chaos Mesh.
 
+Disabled by default. Please don't use it in a production environment unless
+you know the effect explicitly. BTW, although kernel injection will target a
+certain pod, since all pods on the same host share the same kernel, the
+performance of other pods will have some impact, depending on the specific
+callchain and frequency. This feature requires 4.18 kernel or above with
+`CONFIG_BPF_KPROBE_OVERRIDE=y`. To enable it, you need to chage `create` value
```
Comment (cwen0) [inline doc/kernel_chaos.md]: s/`create`/`bpfki.create`/g

File: doc/kernel_chaos.md [hunk 3]
```diff
@@ -2,6 +2,21 @@
 
 This document describe how to add kernel chaos experiments in Chaos Mesh.
 
+Disabled by default. Please don't use it in a production environment unless
```
Comment (cwen0) [inline doc/kernel_chaos.md]: This future is disabled by default.

File: doc/kernel_chaos.md (hunk 2 above)
Comment (cwen0) [inline doc/kernel_chaos.md]: If you want to use kernel chaos, xxxx xxx

File: helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml [hunk 4]
```diff
@@ -64,11 +64,51 @@ spec:
...
+          ports:
+            - name: grpc
+              containerPort: {{ .Values.bpfki.grpcPort }}
+              hostPort: {{ .Values.bpfki.grpcPort }}
+{{- end }}
       volumes:
         - name: socket-path
           hostPath:
             path: {{ .Values.chaosDaemon.socketPath | default "/var/run/docker.sock" }}
+        - name: localtime-path
```
Comment (cwen0) [inline helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml]: if we disabled kernel chaos, these volumes are unnecessary. Can we also use `bpfki.create` field to control these volumes?

File: helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml [hunk 5]
```diff
@@ -64,11 +64,53 @@ spec:
               hostPort: {{ .Values.chaosDaemon.grpcPort }}
             - name: http
               containerPort: {{ .Values.chaosDaemon.httpPort }}
-      {{- with .Values.chaosDaemon.tolerations }}
-      tolerations:
```
Comment (YangKeao) [inline helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml]: Please don't remove it.

Comment (ethercflow) [reply helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml]: @YangKeao Oh, thanks.

**Pull request merged**